elements.html

```diff
@@ -90,15 +91,18 @@ <h5>Überschrift 5</h5>
     </div>
     <div class="container">
         <h1>Heros</h1>
+        .hero--small
+        .hero--color
```

Comment:
Why is this here?
Not necessary

Reply:
Documentational purposes

If you open that page, you can see directly what classes are needed to archive this view.